engines/director/detection_tables.h

```diff
@@ -8950,6 +8951,8 @@ static const DirectorGameDescription gameDescriptions[] = {
 	WINGAME1_l("barbswanlake", "", "SwanLake.exe", "310659620631c126edb6943af1f83e38", 4096651, Common::FR_FRA, 900),
 	WINDEMO1("barbswanlake", "Demo", "SwanLake.exe", "c612aa43e7ef55aa4fd2e1085fb1ef7a", 2385166, 900),
 
+	WINGAME1("barbmagichouse", "", "BARBIE.exe", "11d4eb5c3a76fcb30dadca36ad9d2760", 909919, 400),
```

Comment:
We require tail md5s for projector files.

Reply:
Also, `BARBIE.EXE` is a quite common filename. Any chance of using an additional file for detection?

Reply:
This should be above "barbnail" to maintain alphabetical order, and in the Director v4 section of the file instead of v9

Reply:
OK I think I've addressed all of that. I've also bumped up the game version to 404. Let me know if the formatting of the WINGAME2( macro is ok - a few different approaches in that file.

Reply:
There's another issue... you added the game to the "supported" D4 games but it's not, so it should be among the "unsupported" group. After line 3295, that is.